server/apps/publish/archive_publish.py

```diff
@@ -112,16 +125,26 @@ def update(self, id, updates, original):
             raise SuperdeskApiError.internalError(message="Failed to publish the item: {}"
                                                   .format(str(e)))
 
-    def queue_transmission(self, doc):
+    def publish(self, doc, updates, to_digital_outputs=False):
```

Comment:
Please do return `any_channel_closed` value back to `update()`. Otherwise [SD-1989|https://dev.sourcefabric.org/browse/SD-1989] will reproduce.